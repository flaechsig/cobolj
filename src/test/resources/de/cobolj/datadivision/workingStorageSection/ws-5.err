Bei signed Werten wird ein +/- erwartet