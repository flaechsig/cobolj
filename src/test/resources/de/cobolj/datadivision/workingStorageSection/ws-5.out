My ID is : -12345
